src/schema/__tests__/credit_card.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable promise/always-return */
+import { runQuery } from "test/utils"
+
+describe("CreditCard type", () => {
+  let creditCard = null
+  let rootValue = null
+
+  beforeEach(() => {
+    creditCard = {
+      id: "card123",
+      brand: "Visa",
+      last_digits: "4242",
+    }
+
+    rootValue = {
+      creditCardLoader: sinon.stub().returns(Promise.resolve(creditCard)),
```

Comment:
doesn't need `sinon.stub()`, shd use jest mocking (or since we dont need to assert on args or things like that), can just be `creditCardLoader: () => Promise.resolve(creditCard)`